aiidalab_widgets_base/structures.py

```diff
@@ -287,8 +289,8 @@ def search(self, change=None):  # pylint: disable=unused-argument
             self.start_date = datetime.datetime.strptime(self.date_start.value, '%Y-%m-%d')
             self.end_date = datetime.datetime.strptime(self.date_end.value, '%Y-%m-%d') + datetime.timedelta(hours=24)
         except ValueError:  # Otherwise revert to the standard (i.e. last 7 days)
-            self.start_date = self.dt_end
-            self.end_date = self.dt_now + datetime.timedelta(hours=24)
+            self.start_date = datetime.datetime.now() - datetime.timedelta(days=10)
```

Comment:
Why are 10 days hard-coded here?

Reply:
Just a random default..

Reply:
What are the exact implications? Why not make that a modifiable traitlet and display it in the UI?

Reply:
I started to answer your comment, but then I realized that I had in mind many things to improve. So that is what I did in 5df97c2. I think now things should be more clear.

Answering to your questions: the widgets `self.start_date_widget` and `self.end_date_widget` are there and displayed.

Reply:
So what is happening here: if the user provides the date in a wrong format, it will be replaced by one week range